README.md

```diff
@@ -6,6 +6,8 @@ This page contains a list of example codes written with Optuna.
 ### Simple Black-box Optimization
 
 * [Quadratic function](./quadratic_simple.py)
+- [Quadratic multi-objective function](./quadratic_simple_multi_objective.py)
+- [Quadratic function with constraints](./quadratic_simple_constraint.py)
```

Comment:
This repository already contains examples of multi-objective optimization within the `multi_objective` directory. Would it be possible to move these examples to that location for consistency?

Reply:
I will do it!

Reply:
Thanks!
A nit-picky, but `multi_objective/quadratic_simple_multi_objective.py` might sound verbose since `multi_objective` is duplicated.

Reply:
Addressed!